README.md

```diff
@@ -585,14 +606,17 @@ Keep in mind, that all updates to the NVML bindings code should be made in the
 code be processed and pushed to the `pkg/` directory for release.
 
 ## Releasing
+
 Once the code in `gen/` has been fully updated to support a particular version of NVML, a new release should be created.
 
 As part of the release, two things need to happen:
+
```

Comment:
why the newline?

Reply:
`markdownlint` suggests always having a linebreak before and after lists and code blocks.